app/controllers/talks_controller.rb

```diff
@@ -42,6 +46,10 @@ def talks
     @talks ||= Talk.available
   end
 
+  def recently_given
+    @recent_talks ||= Talk.recently_given
```

Comment:
nitpick: Please name the method and instance variable the same.

Reply:
To clarify, I mean the instance variable and controller method should be the same, the model method is fine.